tests/integration/apps/test_flush.py

```diff
@@ -0,0 +1,78 @@
+import unittest
+from parameterized import parameterized
+import lusid
+import pathlib
+import tests.unit.apps.test_data.test_transactions as txn_test_data
+import lusidtools.apps.flush_transactions as flush
+import datetime
+from dateutil.tz import tzutc
+import logging
+
+
+class FlushAndFillIntegrationTests(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls) -> None:
+        logging.basicConfig(level=logging.INFO)
+        cls.scope = "rjh-test-scope-01"
+        cls.code = "rjhTestPortfolio01"
+        secrets_path = pathlib.Path().resolve().parent.parent / "secrets.json"
+        cls.api_factory = lusid.utilities.ApiClientFactory(
+            api_secrets_filename=secrets_path
+        )
+
+        cls.transaction_portfolios_api = cls.api_factory.build(
+            lusid.api.TransactionPortfoliosApi
+        )
+        response = cls.transaction_portfolios_api.create_portfolio(
```

Comment:
If you stop your test midway though, will this run successfully if the portfolio already exists?